src/charts/stacked-area.js

```diff
@@ -1197,6 +1205,15 @@ define(function(require){
             return this;
         };
 
+        exports.hasOutline = function(_x) {
+            if (!arguments.length) {
+                return hasOutline;
+            }
+            hasOutline = _x;
```

Comment:
Could you explain what the `_x` parameter is?

Reply:
_x is an optional parameter that if passed, will be set as the new ´hasOutline´value, if not, then this function will return the current ´hasOutline´ variable value.

This pattern is effectively a way of having a getter and a setter all at once.
_x could be called anything different for sure, this is the convention we adopted though.

Reply:
Thanks for the explanation! Makes sense, I hadn't seen that convention before.